test/e2e/helm_test.go

```diff
@@ -167,6 +167,20 @@ func TestHelmSetString(t *testing.T) {
 		})
 }
 
+func TestHelmSetFile(t *testing.T) {
+	Given(t).
```

Comment:
good stuff

Reply:
Thanks - it just can't find the files. So I really don't know what I'm missing at this point.

Reply:
@alexec - as I can't get the UI running under Minikube, I'm unable to do any additional debugging at this point to resolve why this test continues to fail. Any suggestions?